copis/config/app.ini

```diff
@@ -0,0 +1,10 @@
+[AppWindow]
+width = 1400
+height = 1000
+
+[Debug]
+env = prod
+
+[Devices]
+items = ./device.ini
+    ./device_1.ini
```

Comment:
Newline at end of file please

Reply:
This doesn't really matter because the app will be modifying this file. But I'll add the new line for now.